runtime/wasm/src/to_from/external.rs

```diff
@@ -344,7 +344,7 @@ impl ToAscObj<AscFullEthereumBlock> for FullEthereumBlockData {
             size: self
                 .size
                 .map(|size| heap.asc_new(&BigInt::from_unsigned_u256(&size)))
-                .unwrap_or_else(|| AscPtr::null()),
+                .into(),
```

Comment:
I was hoping we could go even further here to change...

```rust
size: self
   .size
   .map(|size| heap.asc_new(&BigInt::from_unsigned_u256(&size)))
   .into(),
```
to just...
```rust
size: self.size.into(),
```
Would whatever trait required to do that run afoul of the orphan rules? (If so, a new trait could be added).

Reply:
Hmmm... that would clean up this module, but it obfuscates the conversion to the asc data structure.  Using `into()` makes code navigation more difficult (breaks it in IntelliJ Rust.)

Reply:
I'll leave this for your to decide, but I would have thought that if for example if `u256` had an impl to `AscPtr` that did `heap.asc_new(&BigInt::From_unsigned_u256(value))` then it would be easy to build on top of that to just make anything that was `Option<impl Into<AscPtr>>` call the base into and make it possibly nullable.